container-search/src/main/java/com/yahoo/search/dispatch/searchcluster/Group.java

```diff
@@ -53,25 +55,30 @@ void setHasSufficientCoverage(boolean sufficientCoverage) {
     }
 
     public int workingNodes() {
-        int nodesUp = 0;
-        for (Node node : nodes) {
-            if (node.isWorking() == Boolean.TRUE) {
-                nodesUp++;
-            }
-        }
-        return nodesUp;
+        return (int) nodes.stream().filter(node -> node.isWorking() == Boolean.TRUE).count();
```

Comment:
```suggestion
        return (int) nodes.stream().filter(node -> node.isWorking()).count();
```

Reply:
What happens if 'null' is returned ? Wont that give NullPointerException ?

Reply:
💥